web/Makefile

```diff
@@ -21,9 +18,6 @@ all: changelog dot-github rfc quarto
 
 changelog: $(CHANGELOG_OUTPUT)
```

Comment:
Why keep this target (and the reference above)?

Reply:
No reason to, I overlooked this one.